### ISSUE TITLE
Improve UX of language selection

### ISSUE THREAD

**Issue by boredoms** [Metalab/metalab.at#12]
Currently, the language selection feature is implemented in an ugly way (two <li> items in the header bar with emoji), making the header bar too busy looking.

How can we improve the UX for language selection? My idea is to either right justify or move the language selection elsewhere but not sure where yet.

Any ideas / improvments welcome.

**Comment by anlumo** [comment]
Also, the emojis don't work on Windows.

![image](https://github.com/Metalab/metalab.at/assets/644763/6b137e84-fef1-419d-9335-79007b5867e4)

**Comment by anlumo** [comment]
Reflecting on this, I think it would be ok to move the language selection to the bottom of the page into the footer. Is everyone ok with that?

Many pages have it at the bottom.

**Comment by boredoms** [comment]
yes, i think moving it to the footer is a good idea, if we have too many things in the header we might want to consider a reactive menu and only keep the main n pages on in the header to remove clutter

**Comment by ppiro** [comment]
> Also, the emojis don't work on Windows.

I tested this with Windows 10 22H2 and the flag emoji work fine there.
